src/metabase/driver/postgres.clj

```diff
@@ -892,3 +894,20 @@
 (defmethod driver.sql/default-database-role :postgres
   [_ _]
   "NONE")
+
+(defmethod driver/schema+table->row-count :postgres
+  [driver database]
+  (sql-jdbc.execute/do-with-connection-with-options
+   driver
+   database
+   nil
+   (fn [conn]
+     (try
+      (into {}
+            (map (fn [{:keys [schemaname relname n_live_tup]}]
+                   [[schemaname relname] n_live_tup]))
+            ;; https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-ALL-TABLES-VIEW
+            (next.jdbc/plan conn ["SELECT schemaname, relname, n_live_tup FROM pg_stat_user_tables"]))
```

Comment:
What do you think about using some "as" aliases in the query to make the mapping function read a bit nicer?

e.g. `(fn [{:keys [schema table row_count]}] ...)`